### PR TITLE
Link to the `sizes` parsing steps.

### PR DESCRIPTION
**Pull request by rayankans**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/pull/19.html" title="Last updated on Apr 20, 2020, 12:52 PM UTC (dc0d353)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/image-resource/19/414e2f3...dc0d353.html" title="Last updated on Apr 20, 2020, 12:52 PM UTC (dc0d353)">Diff</a>

### CODE REVIEW

**Comment by rayankans** [comment]
I couldn't figure out the best way to link to the parsing algorithm, since it doesn't seem to be exported. I went ahead and clarified that the parsing steps match the of a `link` tag with `rel=icon`, and linked to the steps in a note.

Is there a better way of doing this? (I'm hoping so)

File: index.html [hunk 1]
```diff
@@ -203,7 +203,12 @@ <h2>
           <ol>
             <li>Set |image|'s [=image resource/sizes=] to the result of parsing
```
Comment (marcoscaceres) [inline index.html:204]: Maybe we should say: 

```suggestion
            <li>Set |image|'s [=image resource/sizes=] to the result of [=link type "icon"|parsing=]
```

Instead... then the parsing algorithm is explicitly linked here and below and we don't need the note.

File: index.html [hunk 2]
```diff
@@ -246,7 +251,12 @@ <h2>
...
             |input|'s "sizes" property value as if it was a [^link/sizes^]
-            attribute. If parsing fails, return failure.
+            attribute of a [^link^] element whose [^link/rel^] attribute
```
Comment (marcoscaceres) [inline index.html:254]: As above...

File: index.html (hunk 1 above)
Comment (rayankans) [reply index.html]: That worked, thanks!

**Pull request merged**